src/Sulu/Component/Content/Document/Subscriber/CopyLocaleSubscriber.php

```diff
@@ -192,8 +191,6 @@ public function checkPageTreeRoute(RoutableBehavior $destDocument, StructureBeha
             'suffix' => $routePath['suffix'],
         ];
         $destDocument->setRoutePath($routePathProp);
-        $destDocument->getRoute()->setPath($routePathProp);
```

Comment:
do we need to call `$destDocument->setRoute(null);`

Reply:
`setRoute` does not accept `null` so we cannot call it